templates/download/shared/_verify-checksums.html

```diff
@@ -1,34 +1,63 @@
-<aside class="p-accordion" role="tablist" aria-multiselect="true" style="border: 1px solid #cdcdcd;">
-  <ul class="p-accordion__list" style="margin-bottom: 0;">
-    <li class="p-accordion__group">
-      <button type="button" class="p-accordion__tab" id="tab1" role="tab" aria-controls="#tab1-section" aria-expanded="false" style="background-color: 0;">Verify your download</button>
-      <section class="p-accordion__panel" id="tab1-section" role="tabpanel" aria-hidden="true" aria-labelledby="tab1-section" style="margin-bottom: 1.5rem; border: 0; background: 0;">
-        <p style="margin-right: 2rem;"><small>Run this command in your terminal in the directory the iso was downloaded to verify the SHA256 checksum:</small></p>
-        <div style="margin-right: 2rem;">
-          <pre style="white-space: pre-wrap;"><code>echo "{{ releases.checksums[system][version] }}" | shasum -a 256 --check</code></pre>
-        </div>
-        <p><small>You should get the following output:</small></p>
-        <pre style="margin-right: 2rem;"><code>ubuntu-{{ version }}-{{ system }}-{{ architecture }}.iso: OK</code></pre>
-        <p><small>Or follow this tutorial to learn <a class="p-link--external" href="https://tutorials.ubuntu.com/tutorial/tutorial-how-to-verify-ubuntu">how to verify downloads</a></small></p>
-      </section>
-    </li>
-  </ul>
-</aside>
+<p>
+  You can
+  <span class="p-contextual-menu--left">
+      <a href="#" class="p-contextual-menu__toggle" aria-controls="#menu-4" aria-expanded="false" aria-haspopup="true">verify your download</a>
+      <span class="p-contextual-menu__dropdown" id="menu-4" aria-hidden="true" aria-label="submenu" style="width: 600px; max-width: 70vw; padding: 1rem;">
+        <span class="p-contextual-menu__group">
+          <small>Run this command in your terminal in the directory the iso was downloaded to verify the SHA256 checksum:</small><br />
+          <code>echo "{{ releases.checksums[system][version] }}" | shasum -a 256 --check</code><br /><br />
+          <small>You should get the following output:</small><br />
+          <code>ubuntu-{{ version }}-{{ system }}-{{ architecture }}.iso: OK</code><br /><br />
+          <small>Or follow this tutorial to learn <a class="p-link--external" href="https://tutorials.ubuntu.com/tutorial/tutorial-how-to-verify-ubuntu">how to verify downloads</a></small>
+        </span>
+      </span>
+  </span>, or get <a href="https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-server" class="p-link--external">help on installing</a>.
```

Comment:
Suggestion: this would point the arrow at the top of the tutorial back to this page. I'm leaving it unversioned so it doesn't trigger a download. What do you think?

```suggestion
  </span>, or get <a href="https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-server?backURL=https://ubuntu.com/download/server/thank-you" class="p-link--external">help on installing</a>.
```

Reply:
@anthonydillon ^